integration-tests/integration/reportPages/change-prison.spec.js

```diff
@@ -0,0 +1,118 @@
+const ReportUseOfForcePage = require('../../pages/createReport/reportUseOfForcePage')
+const IncidentDetailsPage = require('../../pages/createReport/incidentDetailsPage')
+const ChangePrisonPage = require('../../pages/createReport/changePrisonPage')
+const CheckAnswersPage = require('../../pages/createReport/checkAnswersPage')
+const { ReportStatus } = require('../../../server/config/types')
+
+context('Submitting details page form', () => {
+  const bookingId = 1001
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubLogin')
+    cy.task('stubOffenderDetails', bookingId)
+    cy.task('stubLocation', '357591')
+    cy.task('stubLocations', 'MDI')
+    cy.task('stubLocations', 'LEI')
+    cy.task('stubPrison', 'MDI')
+    cy.task('stubPrison', 'LEI')
+    cy.task('stubPrisons')
+    cy.task('stubOffenders')
+    cy.task('stubUserDetailsRetrieval', 'MR_ZAGATO')
+    cy.task('stubUserDetailsRetrieval', 'MRS_JONES')
+    cy.task('stubUserDetailsRetrieval', 'TEST_USER')
+    cy.task('seedReport', { status: ReportStatus.IN_PROGRESS })
+  })
+
+  const fillFormAndSave = () => {
+    const reportUseOfForcePage = ReportUseOfForcePage.visit(bookingId)
+    const incidentDetailsPage = reportUseOfForcePage.startNewForm()
+    incidentDetailsPage.offenderName().contains('Norman Smith')
+    incidentDetailsPage.location().select('Asso A Wing')
+    incidentDetailsPage.forceType.check('true')
+    incidentDetailsPage.save()
+  }
+
+  const completeIncidentDetails = () => {
+    cy.login(bookingId)
+    fillFormAndSave()
+    cy.go('back')
+  }
+
+  it('Can edit prison prior to reaching check-your-answers page', () => {
+    completeIncidentDetails()
+
+    const incidentDetailsPage = IncidentDetailsPage.verifyOnPage()
+
+    incidentDetailsPage.clickChangePrison()
+
+    const changePrisonPage = ChangePrisonPage.verifyOnPage()
+
+    changePrisonPage.changePrisonToLeeds()
```

Comment:
would parameterise this, like we do with location below: `    incidentDetailsPage.location().select('Asso A Wing')`

Reply:
done